src/main/java/com/conveyal/datatools/manager/controllers/api/ProjectController.java

```diff
@@ -306,7 +304,11 @@ private static Project thirdPartySync(Request req, Response res) {
 
         LOG.info("syncing with third party " + syncType);
         if(DataManager.feedResources.containsKey(syncType)) {
-            DataManager.feedResources.get(syncType).importFeedsForProject(proj, req.headers("Authorization"));
+            try {
+                DataManager.feedResources.get(syncType).importFeedsForProject(proj, req.headers("Authorization"));
+            } catch (Exception e) {
+                haltWithMessage(req, 500, "An error occurred while trying to sync", e);
```

Comment:
Isn't this PR partially intended to remove generic exceptions? Does this hinder its purpose?

Reply:
The most important part of this PR is the creation of bugsnag reports whenever a 500 error halt occurs.  Wherever possible I think it's good to be as specific as possible to create better error reports, but I think it's fine to catch generic exceptions too.  In this specific case, the abstract class method only specifies the Exception class as something that is thrown.